plugins/versionpress/src/Utils/QueryLanguageUtils.php

```diff
@@ -83,7 +83,7 @@ public static function entityMatchesSomeRule($entity, $rules)
 
                 if ($isWildcard && preg_match(QueryLanguageUtils::tokensToRegex($valueTokens), $entity[$field])) {
                     return true;
-                } elseif (strval($entity[$field]) === $value) {
+                } elseif (strtolower(strval($entity[$field])) === strtolower($value)) {
```

Comment:
`$value` will be lowercased automatically because of this: https://github.com/versionpress/versionpress/blob/4475b80e305d3ca46bff7b3770a5e36ef56fc6a5/plugins/versionpress/src/Utils/QueryLanguageUtils.php#L47-L49

Reply:
Thanks! I updated this PR to remove the unnecessary `strtolower` that I added.